plaso/parsers/utmp.py

```diff
@@ -162,13 +162,22 @@ def ParseFileObject(self, parser_mediator, file_object):
           'Unable to parse first utmp entry with error: {0!s}'.format(
               exception))
 
+    if not timestamp:
```

Comment:
since both username and hostname can be legitimately empty do the following:

if the `type of login` of the first record is 0x00000001 (RUN_LVL) or 0x00000002 (0x00000002)
* username is not empty and in "shutdown", "reboot", "runlevel"
* terminal should be "~"

if the `type of login` is 0 (EMPTY)
* all should be empty?

Reply:
I thought about this a bit more, and realized the real issue here was the decode errors being caught and emitted as warnings in _ReadEntry. I've changed this around, but I don't think we need to add more validation here just yet.